src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseEventTrigger.java

```diff
@@ -103,4 +105,12 @@ public String getEndButtonLabel() {
     public void setEndButtonLabel(String endButtonLabel) {
         this.endButtonLabel = endButtonLabel;
     }
+
+    public Boolean getCanSaveDraft() {
```

Comment:
the java convention for bool methods is to start with `is`. we should rename this to `isCanSaveDraft`?
which doesn't sound that nice. I think we should rename this to `isDraftEnabled`

Reply:
I think this is consistent with all other Boolean type variables in this class. Also not sure jackson accepts is... vs get... methods.

Reply:
Jackson can handle bool methods starting with is